### ISSUE TITLE
i was install your anbox-direct-gpu-access....

### ISSUE THREAD

**Issue by weihongman** [elliott-wen/anbox-direct-gpu-access#5]
hi elliott!
i was deploy ubuntu 18.04 and install anbox-module.
and run emugui and lxc-start -F -n andriod command.
lxc container status running. but emugui show not anything.
blank android windows is show.
i wondering this is it? or something wrong?
also lxc start command after into container elliott shell. process adbd running but adb command not found.
plz tell me why.. 
thanks
![container1](https://user-images.githubusercontent.com/63442339/78969443-e8614180-7b41-11ea-8d39-a1fa2e0a3b8d.png)
![container2](https://user-images.githubusercontent.com/63442339/78969447-e9926e80-7b41-11ea-9c4c-abccc8624ec4.png)

![app_window](https://user-images.githubusercontent.com/63442339/78969492-05961000-7b42-11ea-8ccc-1d0922c9412d.png)

**Comment by Genxster1998** [comment]
Are you running it on intel graphics device ,adb is to be installed to system

**Comment by elliott-wen** [comment]
Sorry for the late reply. Having busy days recently.

I guess you need to figure out your container IP first.

after that, use adb connect IP to establish the connection.

This project is just for fun, may contain bugs.

**Comment by elliott-wen** [comment]
Yes. I am running with on intel CPU built-in graphics.

If you wanna use adb, you should figure out the IP address of your
container. And use
adb connect IP

**Comment by Genxster1998** [comment]
@elliott-wen can you share your rootfs source changes/patches/commits  ?

**Comment by elliott-wen** [comment]
Yes. I am very much willing to. I basically redo what anbox does and you can find most of the changes in their github repos. (Especially in their android system repos.

I need to search the code in the computer at my office and will keep you updated when I find it.

**Comment by Genxster1998** [comment]
@elliott-wen Thanks Sir , I am basically more interested in graphics stack u are using like android-x86 to direct expose GPU to container

**Comment by elliott-wen** [comment]
Let me recall. I think I was using mesa for 3D acceleration and minigbm for
buffer allocation.

**Comment by elliott-wen** [comment]
I guess modern Android also supports SwiftSharder, which requires no GPU
interaction.

**Comment by Genxster1998** [comment]
@elliott-wen yeah sir, are you releasing your rootfs commits related to graphics and audio/media stack ,as anbox lacks it but I am also interested in spurv , https://www.collabora.com/news-and-blog/blog/2017/11/27/running-chromium-with-ozone-gbm-on-gnu-linux/ and chrome exosphere Wayland implementation for graphics(too smooth + multiwindow).Anbox is not stable at all ,close button unresponsive sometimes as host wm unresponsive to container action ,screen buffer update issue etc .
Have a great day.

**Comment by Genxster1998** [comment]
@elliott-wen can you hint me how did you setup 
Sockets to configure input and gps ,alsaout at host . Btw adbd server not running as eth0 interface in container can't allocate ip address.
Also , can you hint me to have resizable window ?   Also have you made any other changes in grallocgbm for 900*640 screen only .
